cms/templates/js/mock/mock-settings-page.underscore

```diff
@@ -62,8 +62,8 @@
         <span class="tip tip-stacked timezone">(UTC)</span>
       </div>
     </li>
-<<<<<<< HEAD
-=======
+<!-- <<<<<<< HEAD -->
```

Comment:
As per response from CourseOps in Slack, none of our courses should be using "course prerequisites" so removing this as per HEAD is fine.